packages/cashier/src/pages/withdrawal/crypto-withdraw-form/crypto-withdraw-form.tsx

```diff
@@ -190,7 +189,6 @@ const CryptoWithdrawForm = observer(() => {
                     </form>
                 )}
             </Formik>
-            <MobileWrapper>{isCryptocurrency(currency) ? <RecentTransaction /> : null}</MobileWrapper>
```

Comment:
No need to render side notes separately in mobile mode inside child components as it will now be handled by `<PageContainer />`